arch/unitroot/unitroot.py

```diff
@@ -1714,3 +1714,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+  def BandWidthNW(y, kernel = "ba"):
```

Comment:
Should be `autobandwitdh` or `auto_bandwidth` (probablye the latter).

So space around = for default args.